src/github/pullRequest.ts

```diff
@@ -102,4 +104,53 @@ export class PullRequestApi {
   getAuthor(): string {
     return this.pr.user.login;
   }
+
+  /**
+   * Generates a Check Run or modifies the existing one.
+   * This way we can aggregate all the results from different causes into a single one
```

Comment:
Does it mean that you have solved the problem of two status checks - from `pull_request` and from `pull_request_review` triggers?

Reply:
Kind of yes. you will still find two triggers if it is triggered from both events. 

But now they create an extra check which contains the final result, so we only use that one as the blocker.